packages/flutter_tools/lib/src/test/flutter_web_platform.dart

```diff
@@ -186,14 +205,18 @@ class FlutterWebPlatform extends PlatformPlugin {
     if (request.url.path.endsWith('.dart.browser_test.dart.js')) {
       final String leadingPath = request.url.path.split('.browser_test.dart.js')[0];
       final String generatedFile = _fileSystem.path.split(leadingPath).join('_') + '.bootstrap.js';
-      return shelf.Response.ok(bootstrapFileContents('/' + generatedFile, 'require.js', 'dart_stack_trace_mapper.js'), headers: <String, String>{
+      return shelf.Response.ok(generateTestBootstrapFileContents('/' + generatedFile, 'require.js', 'dart_stack_trace_mapper.js'), headers: <String, String>{
```

Comment:
> Flutter run and test can share the generated bootstrap module.

This looks like it's doing the same thing in the same spot.  Will there be a follow up PR where you use the bootstrap code in a second place? Or am I missing it.

Reply:
generatedActualMain was deleted and replaced with the existing logic in web/bootstrap.dart. bootstrapFileContents was renamed to generateTestBootstrapFileContents and moved to bootstrap.dart but is otherwise unchanged

Reply:
Ah I missed that `generateMainModule` wasn't new.